### ISSUE TITLE
Site.LanguageCode should consider .Site.Language first

### ISSUE THREAD

**Issue by tdelmas** [gohugoio/hugo#4892]
https://github.com/gohugoio/hugo/blob/master/docs/config.toml#L7
```
languageCode = "en-us"
```

Sould be there:
https://github.com/gohugoio/hugo/blob/master/docs/config.toml#L280
```
[languages]
  [languages.en]
    contentDir = "content/en"
    languageName = "English"
    languageCode = "en-us"
```

It impacts for example the language displayed in the RSS feed:
https://github.com/gohugoio/hugo/blob/master/docs/layouts/index.rss.xml#L7
```{{ with .Site.LanguageCode }}<language>{{.}}</language>{{end}}```

The documentation, particularity https://gohugo.io/content-management/multilingual/#configure-languages should be updated too.

**Comment by Bin95** [comment]
> **Site.LanguageCode should consider .Site.Languate first**   
**.Site.Languate** should be a  typo of **.Site.Language**?

**Comment by tdelmas** [comment]
The issue is still there. With the typo reported by @Bin95 .

**Comment by jmooring** [comment]
This is a duplicate of https://github.com/gohugoio/hugo/issues/10690, which was resolved in v0.112.5. The documentation will be updated in https://github.com/gohugoio/hugoDocs/pull/2079.
